rotkehlchen/tests/api/test_blockchain.py

```diff
@@ -1937,52 +1926,38 @@ def test_remove_ksm_blockchain_account(rotkehlchen_api_server):
     """
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
     async_query = random.choice([False, True])
-
-    setup = setup_balances(
-        rotki=rotki,
-        ethereum_accounts=None,
-        btc_accounts=None,
-        eth_balances=None,
-        token_balances=None,
-        btc_balances=None,
+    response = requests.get(api_url_for(
+        rotkehlchen_api_server,
+        "blockchainbalancesresource",
```

Comment:
keep your eyes open for these stuff please

```suggestion
        'blockchainbalancesresource',
```